src/source_distribution.rs

```diff
@@ -551,20 +553,36 @@ pub fn source_distribution(
         }
     }
 
+    let mut include = |pattern| -> Result<()> {
+        println!("📦 Including files matching \"{}\"", pattern);
+        for source in glob::glob(&pyproject_dir.join(pattern).to_string_lossy())
+            .expect("No files found for pattern")
+            .filter_map(Result::ok)
+        {
+            let target = root_dir.join(source.strip_prefix(pyproject_dir).unwrap());
+            if source.is_dir() {
+                writer.add_directory(target)?;
+            } else {
+                writer.add_file(target, source)?;
+            }
+        }
+        Ok(())
+    };
+
+    #[allow(deprecated)]
     if let Some(include_targets) = pyproject.sdist_include() {
+        warn!("`[tool.maturin.sdist-include]` is deprecated, please use `[tool.maturin.include]`");
```

Comment:
`warn!` requires settings `RUST_LOG` and will add timestamp to the message, just use `eprintln!` instead.